File: src/net/rizon/moo/tinc/tinc.scala
package net.rizon.moo.tinc

import net.rizon.moo.mpackage

object tinc
{
	def tincBase = ".tinc"
	var layers: List[layer] = List(new layer("tl"), new layer("rc"), new layer("rh"))
}

class tinc extends mpackage("tinc", "Manages tinc")
{
	new commandTinc(this)
}

File: src/net/rizon/moo/tinc/tinc.java
package net.rizon.moo.tinc;

import net.rizon.moo.MPackage;

public class tinc extends MPackage
{
	public tinc()
	{
		super("tinc", "Manages tinc");
		
		new CommandTinc(this);
	}
	
	public static final String tincBase = ".tinc";
	protected static Layer[] Layers = new Layer[] { new Layer("tl"), new Layer("rc"), new Layer("rh") };
}